### ISSUE TITLE
[BITV] 9.1.4.11/2.3 - "Weather" menu - The "Favorites" button does not meet the contrast requirements for graphical controls, whether it is active or inactive. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36969]
- [ ] Fix
- [ ] Backport

![20ff47cad6376bf69aae1fdaf0a50b3e](https://user-images.githubusercontent.com/1315170/222241446-d86d452a-d42a-4902-9f08-070ac0b93081.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3dbfb7615a-v2-n3

</details>
